bank_api_integration/bank_api_integration/doctype/bulk_outward_bank_payment/bulk_outward_bank_payment.py

```diff
@@ -36,14 +36,21 @@ def validate(self):
 	def on_submit(self):
 		if self.workflow_state == 'Approved':
 			failed_obp_list = []
+			password_entered = None
 			integration_doc = frappe.get_doc('Bank API Integration', {'bank_account': self.company_bank_account})
+			disabled_accounts = frappe.get_site_config().bank_api_integration['disable_transaction']
+			
+			if disabled_accounts == '*' or integration_doc.account_number in disabled_accounts:
+				frappe.throw(_(f'Unable to process transaction for the selected bank account. Please contact Administrator.'))
+				return
 
-			password_defined = integration_doc.get_password(fieldname="transaction_password")
-			password_entered = self.get_password(fieldname="transaction_password")
+			if integration_doc.enable_password_security:
+				password_defined = integration_doc.get_password(fieldname="transaction_password")
+				password_entered = self.get_password(fieldname="transaction_password")
```

Comment:
please get the password in a pop-up along with the top if it is applicable. storing it in the document is not so useful...

Reply:
Dialog updated here https://github.com/aerele/bank_api_integration/pull/3/commits/402c4f8730b184bbafda9ac023dea4c9c8cdb63a